prepare_data.py

```diff
@@ -1,4 +1,18 @@
 import pandas as pd
+import json
+import re
+from nltk.tokenize import word_tokenize
+from sklearn.feature_extraction.text import TfidfVectorizer
+from scipy.sparse import hstack
+from sklearn.preprocessing import LabelEncoder
+import subprocess
+
+# Function to run another Python script
+def run_script(script_name):
+    subprocess.check_call(['python', script_name])
```

Comment:
@SnehaHS65 This code is not working for me:

```python
pip install -r requirements.txt; python prepare_data.py
```

```python
  File "prepare_data.py", line 12, in run_script
    subprocess.check_call(['python', script_name])
  File "subprocess.py", line 413, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['python', 'download_nltk_data.py']' returned non-zero exit status 1.
```

Can we just use `import download_nltk_data` instead?